addons/website_slides/controllers/main.py

```diff
@@ -584,9 +584,11 @@ def slide_like(self, slide_id, upvote):
             return fetch_res
         # check slide operation
         slide = fetch_res['slide']
+        if not slide.channel_id.is_member:
+            return {'error': 'not_member'}
```

Comment:
In survey we used some prefix to better understand error keys. It is not standard but I think it helps, like
```
channel_membership_required
channel_comment_disabled
channel_karma_required
```